build/dependencies.props

```diff
@@ -8,6 +8,7 @@
     <MicrosoftSourceLinkGitHubVersion>1.0.0</MicrosoftSourceLinkGitHubVersion>
     <NetStandardVersion>2.0.3</NetStandardVersion>
     <NewtonsoftVersion>13.0.3</NewtonsoftVersion>
+    <SystemDiagnosticSourceVersion>7.0.2</SystemDiagnosticSourceVersion>
```

Comment:
7.0.2 because it is the lowest version with a 462 target?

Reply:
I meant to have the latest here. fixed